apps/web/src/stores/evaluationResultsWithMetadata.ts

```diff
@@ -18,34 +17,33 @@ export default function useEvaluationResultsWithMetadata(
     commitUuid: string
     projectId: number
   },
-  opts: SWRConfiguration,
+  { fallbackData }: SWRConfiguration = {},
 ) {
   const { toast } = useToast()
-  const { data = EMPTY_ARRAY, ...rest } = useSWR<
-    EvaluationResultWithMetadata[]
-  >(
-    ['evaluationResults', evaluationId, documentUuid, commitUuid, projectId],
-    async () => {
-      const [data, error] = await computeEvaluationResultsWithMetadataAction({
-        evaluationId,
-        documentUuid,
-        commitUuid,
-        projectId,
-      })
+  const fetcher = useCallback(async () => {
+    const [data, error] = await computeEvaluationResultsWithMetadataAction({
+      evaluationId,
+      documentUuid,
+      commitUuid,
+      projectId,
+    })
 
-      if (error) {
-        toast({
-          title: 'Error fetching evaluations',
-          description: error.formErrors?.[0] || error.message,
-          variant: 'destructive',
-        })
-        throw error
-      }
+    if (error) {
+      toast({
+        title: 'Error fetching evaluations',
+        description: error.formErrors?.[0] || error.message,
+        variant: 'destructive',
+      })
```

Comment:
nice toast should we also use toast in the other fetchers you've implemented in case of error?